File: qqbot-admin-api/src/main/java/com/mieai/qqbot/admin/events/InboxEventSummaryResponse.java
package com.mieai.qqbot.admin.events;

import java.time.Instant;
import java.util.UUID;

/** Public, non-payload representation used by the Inbox list. */
public record InboxEventSummaryResponse(
        UUID id,
        String environment,
        UUID botId,
        String botDisplayName,
        String appId,
        String eventType,
        String platformEventId,
        String status,
        int attempt,
        Instant availableAt,
        Instant receivedAt,
        Instant updatedAt,
        String lastError) {
}

File: qqbot-module-operations/src/main/kotlin/com/mieai/qqbot/admin/events/InboxEventSummaryResponse.kt
package com.mieai.qqbot.admin.events

import java.time.Instant
import java.util.UUID

/** Public, non-payload representation used by the Inbox list. */
data class InboxEventSummaryResponse(
    val id: UUID,
    val environment: String,
    val botId: UUID,
    val botDisplayName: String?,
    val appId: String?,
    val eventType: String,
    val platformEventId: String,
    val status: String,
    val attempt: Int,
    val availableAt: Instant,
    val receivedAt: Instant,
    val updatedAt: Instant,
    val lastError: String?,
)
